places/hunter.md

```diff
@@ -0,0 +1,5 @@
+## Hunter, KS
+
+### Great Places to Eat
+
+### Fun things to do
```

Comment:
> Is there anything fun to do in Hunter? I've heard it's only OK :cry:

There is a small cafe here, that's about it. Not really a destination for anything except hunting.